courses/machine_learning/deepdive2/structured/solutions/4a_sample_babyweight.ipynb

```diff
@@ -0,0 +1,2808 @@
+{
```

Comment:
<p>If you do floor division (//), you won't have to do all the casting later</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/671/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I wanted to make it more clear to newer students what I am doing since they usually are a lot less familiar with floor division.</p>

Reply:
Makes sense. If that's the case, you can do the casting here to avoid repeatedly doing it later.

Reply:
<p>I want to leave the casting below in create_data_split_sample_df function so that students can see locally the cumulative sum of splits happening there and not way up top. It should provide a better intuition for them on how the data is distributed.</p>